dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassPathBuilderTest.java

```diff
@@ -184,26 +184,4 @@ public void testArtifactToClasspath_notToGenerateRepositoryException()
     List<Path> paths = ClassPathBuilder.artifactsToClasspath(ImmutableList.of(jamonApiArtifact));
     Truth.assertThat(paths).isNotEmpty();
   }
-
-  @Test
-  public void testArtifactToClasspath_reportAggregatedRepositoryException()
```

Comment:
This test has been removed because now `jamonapi` can build a class path thanks to the change https://github.com/GoogleCloudPlatform/cloud-opensource-java/pull/365#discussion_r250581145